alerts/src/iosMain/kotlin/AlertPresenter.kt

```diff
@@ -119,20 +120,26 @@ actual class AlertPresenter(
             alert.message,
             transform(alert.style),
         ).apply {
+            fun Alert.Action.isCancelAction(): Boolean =
+                (this.style == Alert.Action.Style.CANCEL) or (this.style == Alert.Action.Style.NEGATIVE)
             alert.actions.forEach { action ->
                 addAction(
                     UIAlertAction.actionWithTitle(
                         action.title,
                         transform(action.style),
                     ) {
-                        logger.info(TAG, "Action ${action.title} was called on dialog with title: ${alert.title}")
+                        if (action.isCancelAction()) {
+                            logger.info(TAG, "Dismissing alert dialog with title: ${alert.title}")
```

Comment:
You dont know this, a custom action may have a negative or cancel style. Also all actions dismiss

Reply:
You are right, I will adjust to log dismissal here. I was trying to figure out a way to add an onDismiss callback but I could not find any. If you know where to point me I think that would be a better solution.

Reply:
Changed to that but I see issue with logs not called. For Android dismiss logs are not called when dismissing via action. On iOS appear / dismiss logs are not displayed but click action logs are. Will investigate more.

Reply:
I think it looks ok, logs for iOS:

Displaying alert dialog with title: Hello, Kaluga 🐟
10-11 18:28:14.067 💙 INFO AlertDialog - Action Cancel was called on dialog with title: Hello, Kaluga 🐟
10-11 18:28:15.466 💙 INFO AlertDialog - Displaying alert dialog with title: Wait for 3 sec...
10-11 18:28:18.483 💙 INFO AlertDialog - Dismissing alert dialog with title: Wait for 3 sec...
10-11 18:28:20.165 💙 INFO AlertDialog - Displaying alert dialog with title: Wait for 3 sec...
10-11 18:28:21.520 💙 INFO AlertDialog - Action OK was called on dialog with title: Wait for 3 sec...
10-11 18:28:23.432 💙 INFO AlertDialog - Displaying alert dialog with title: Select an option
10-11 18:28:24.990 💙 INFO AlertDialog - Action Option 3 was called on dialog with title: Select an option

And for Android:

Displaying alert dialog with title: Hello, Kaluga 🐟
Action OK was called on dialog with title: Hello, Kaluga 🐟
Displaying alert dialog with title: Hello, Kaluga 🐟
Action Cancel was called on dialog with title: Hello, Kaluga 🐟
Displaying alert dialog with title: Wait for 3 sec...
Dismissing alert dialog with title: Wait for 3 sec...
Displaying alert dialog with title: Wait for 3 sec...
Action OK was called on dialog with title: Wait for 3 sec...
Displaying alert dialog with title: Select an option
Action Option 2 was called on dialog with title: Select an option
Displaying alert dialog with title: Hello, Kaluga 🐟
Action OK was called on dialog with title: Hello, Kaluga 🐟
Displaying alert dialog with title: Hello, Kaluga 🐟
Action Cancel was called on dialog with title: Hello, Kaluga 🐟
Displaying alert dialog with title: Wait for 3 sec...
Dismissing alert dialog with title: Wait for 3 sec...
Displaying alert dialog with title: Wait for 3 sec...
Action OK was called on dialog with title: Wait for 3 sec...
Displaying alert dialog with title: Select an option
Canceling alert dialog with title: Select an option
Displaying alert dialog with title: Select an option
Action Option 3 was called on dialog with title: Select an option